.github/workflows/ci.yml

```diff
@@ -24,12 +24,15 @@ jobs:
   test_ci:
     runs-on: ubuntu-latest
     name: test_ci
+    strategy:
+      matrix:
+        java-version: [ 11, 17 ]
```

Comment:
In addition to running the CI with both java version it would be great to somehow run e2e-s as well on different versions.

That may be a bit trickier because we would have to change the docker base image etc. What do you think? 
If it's a lot of work we can open a follow-up ticket for that as well , but we need that before we can say java 17 is supported

Reply:
I made it for my own fork, so it should not be complecated.

The only thing that stopped me from adding this to the PR is that after that there are 200+ jobs running in ci... and the total time is more than an hour

If it's ok, then i can add it here as well

Reply:
We should not run it for all jobs. We can target only the latest Flink version (1.18) for now

Reply:
ok, now done for 1.18 only